java/src/main/java/com/google/cloud/dataproc/templates/util/TemplateConstants.java

```diff
@@ -503,4 +503,10 @@ public interface TemplateConstants {
   String KAFKA_BQ_CONSUMER_GROUP_ID = "kafka.bq.consumer.group.id";
 
   String KAFKA_BQ_BATCH_INTERVAL = "kafka.bq.batch.interval";
+
+  /** Kafka To GCS - Direct stream properties */
+  String KAFKA_CONSUMER_GROUP_ID = "kafka.gcs.consumer.group.id";
+
+  String KAFKA_BATCH_INTERVAL = "kafka.gcs.batch.interval";
```

Comment:
If a property is only specific to a template, please use naming conventions like: KAFKA_GCS_*. See if this can be reused accros Kafka templates, if not please rename it to KAFKA_GCS_*

Reply:
Done